ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -7872,3 +7877,56 @@ module VTPM = struct
     let ref = Client.VTPM.get_by_uuid ~rpc ~session_id ~uuid in
     Client.VTPM.destroy ~rpc ~session_id ~self:ref
 end
+
+module Tracing = struct
+  let create _printer rpc session_id params =
+    let name_label = List.assoc "name-label" params in
+    let hosts =
+      List.assoc_opt "hosts" params
+      |> Option.fold ~none:[] ~some:(fun hosts ->
+             List.map
+               (fun uuid -> Client.Host.get_by_uuid ~rpc ~session_id ~uuid)
+               (String.split_on_char ';' hosts)
+         )
+    in
+    let status =
+      List.assoc_opt "status" params
+      |> Option.fold ~none:false ~some:(fun s ->
+             try Stdlib.bool_of_string s with _ -> false
+         )
+    in
+    let tags =
+      List.assoc_opt "tags" params
+      |> Option.fold ~none:["bugtool"] ~some:(String.split_on_char ';')
+      |> List.filter_map (fun kv ->
+             match String.split_on_char ':' kv with
+             | [k; v] ->
+                 Some (k, v)
+             | _ ->
+                 None
+         )
+    in
+    let endpoints =
+      List.assoc_opt "endpoints" params
+      |> Option.fold ~none:["bugtool"] ~some:(String.split_on_char ';')
```

Comment:
The fact that something is split on semicolon should probably be mentioned in the documentation above. Is splitting on semicolon standard? Xapi often uses a comma.

Reply:
Resolved by 
0b2b538